content/intro-to-storybook/react-native/en/simple-component.md

```diff
@@ -35,7 +35,9 @@ module.exports = {
 
 If you check the `stories` property you'll see that Storybook is looking for stories in the `components` folder.
 
-In React Native Storybook we use the config in `main.js` to generate a file called `storybook.requires.js`, this is because React Native doesn't support dynamic imports yet. This file gets generated when you run `yarn storybook` to start Storybook or `yarn storybook-generate` if you just want to regenerate the `storybook.requires.js` file. This file is used to load all the stories in the project and it also imports your addons, whenever you find that a story is not being loaded you can try regenerating this file.
+In React Native Storybook we use the config in `main.js` to generate a file called `storybook.requires`, this is due to some limitations with the metro bundler. This file gets generated when you run `yarn storybook` to start Storybook or `yarn storybook-generate` which just regenerates the `storybook.requires` file. This file is used to load all the stories in the project and it also imports your addons, whenever you find that a story is not being loaded you can try regenerating this file. Since v7 we are now able to dynamically import new stories that match your config so you shouldn't need to regenerate this file unless you change your `main.js` config.
+
+You'll also find the generate function in `metro.config.js` that will run the generation whenever you start metro.
```

Comment:
@dannyhw we have a bit of room for improvement here. What do you think about rephrasing this as follows:

```suggestion
In Storybook for React Native, due to current limitations with the Metro bundler, we rely on the configuration in `main.js` to generate a file called `storybook.requires`, which is used to load all our stories and addons in our project. This file is automatically generated when you run `yarn storybook` to start your Storybook.

<div class="aside">

💡 You can also manually generate the `storybook.requires` file by running `yarn storybook-generate`. However, you shouldn't need to re-create this file unless you see that a story is not being loaded or notice that a change in your `main.js` config is not reflected in your Storybook. To learn more about how the `storybook.requires` file is generated, you can check out the `generate` function in your `metro.config.js` file.

</div>
```

With it, we can share the same information without going too deep into the specifics nd keep the momentum going in terms of flow.